testplan/common/entity/base.py

```diff
@@ -1123,21 +1143,15 @@ class RunnableManager(Entity):
     :py:class:`Runnable <testplan.common.entity.base.Runnable>` entity
     in a separate thread and handles the abort signals.
 
-    :param parse_cmdline: Parse command lne arguments.
+    :param parse_cmdline: Parse command line arguments.
     :type parse_cmdline: ``bool``
```

Comment:
now the parser argument is no more in the docs, is it intentional?

Reply:
The original doc string has argument `resources`, `runner` and `parser`, but they don't exist in config options. I add them back, but seems only `resources` and `runner` are reasonably needed here, `parser` is not necessary in a `RunnableManager`, so is `parse_cmdline`. I would like to move them to `Testplan` class, is that OK ?

Reply:
Finally I decide to remove `parser` from TestRunner config, or it will make code complicated. Because we need to reserve the `_default_options` (which is enriched) for parser.